.github/workflows/backend.yml

```diff
@@ -107,6 +107,51 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-in-silo-modes:
+    if: needs.files-changed.outputs.backend == 'true'
+        && (
+             contains(github.ref, '/hc-')
+          || contains(github.ref, 'hybrid-cloud')
```

Comment:
Where as this catches branches that refer to general hybrid cloud.

Reply:
Ideally we'd have one pattern, but I legitimately see it common for developers to either cut branches in accordance to a jira ticket, OR some hybrid cloud broad task.  We can communicate to the team that they have either option.